Activities/AreaEditor.cpp

```diff
@@ -381,8 +381,6 @@ void AreaEditor::Update()
 			Activity * pActivity = dynamic_cast<Activity *>(pActivityPreset->Clone());
 			GameActivity *pTestGame = dynamic_cast<GameActivity *>(pActivity);
             RTEAssert(pTestGame, "Couldn't find the \"Skirmish Defense\" GAScripted Activity! Has it been defined?");
-            pTestGame->SetPlayerCount(1);
```

Comment:
Why did you kill the SetPlayerCount and SetTeamCount here and in SceneEditor.cpp? Does this just set them to default values or something?

Reply:
The setters did nothing, probably for a while now.
Player and Team counts are automagic now so using setters/properties doesn't affect anything so I just removed them.